bindata/network/ovn-kubernetes/007-flowschema.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: flowcontrol.apiserver.k8s.io/v1alpha1
```

Comment:
this must not use v1alpha1. v1alpha1 is subject to going away. Please switch to v1beta1 asap.

Reply:
OK, I will file a PR momentarily for that. Thanks for letting us now

Reply:
@alexanderConstantinescu based on the discussion with @sttts we should leave the version to `v1alpha` for 4.7. But we should set the version to `v1beta` in 4.8. Please see https://bugzilla.redhat.com/show_bug.cgi?id=1913399#c1